GTG/gtk/browser/browser.py

```diff
@@ -1066,13 +1139,16 @@ def on_dismiss_task(self, widget):
                 task.set_status(Task.STA_DISMISSED)
                 self.close_all_task_editors(uid)
 
-    def apply_filter_on_panes(self, filter_name, refresh=True):
+    def apply_filter_on_panes(self, filter_name, refresh=True,
+                              parameters=None):
         """ Apply filters for every pane: active tasks, closed tasks """
         # Reset quickadd_entry if another filter is applied
         self.quickadd_entry.set_text("")
         for pane in self.vtree_panes:
             vtree = self.req.get_tasks_tree(name=pane, refresh=False)
-            vtree.apply_filter(filter_name, refresh=refresh)
+            vtree.apply_filter(filter_name, refresh=refresh,
+                               parameters=parameters)
+            vtree.refresh_all()
```

Comment:
Please, create a PR for liblarch that fixes the refresh logic when parameters are specified.